builds/misc/images-mqtt.yaml

```diff
@@ -0,0 +1,57 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - master
+pr: none
+
+jobs:
+################################################################################
+  - job: linux_amd64
+################################################################################
+    displayName: Linux AMD64
+    pool:
+      vmImage: 'ubuntu-latest'
+
+    variables:
+      imageName: $(registry.address)/azureiotedge/mqttd-linux-amd64
+
+    steps:
+      - bash: 'docker login $(registry.address) --username $(registry.user) --password $(registry.password)'
```

Comment:
https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/build/docker?view=azure-devops#login

Reply:
Good point. I followed what we had in other pipelines, but using the fit-for-purpose task is better. I also don't see an existing service connection in Azure DevOps for EdgeBuilds container registry (@philipktlin do you have one?). I'll create one if we don't have.

Reply:
please go ahead to create one. :)